war/src/main/java/io/lumeer/engine/controller/configuration/ConfigurationManipulator.java

```diff
@@ -150,8 +127,8 @@ public void resetConfiguration(final String collectionName, final String nameVal
     * @param value
     *       the Object value of the given key
     */
-   public void setConfiguration(final String collectionName, final String nameValue, final String key, final Object value) {
-      writeValueToDb(collectionName, nameValue, key, value);
+   public void setConfiguration(final String collectionName, final String nameValue, final String key, final Object value, final Map<String, Boolean> flags) {
```

Comment:
Wouldn't be flags in form Set<String> be better? Allow null flags and an overloaded method variant without flags (assuming empty set).